.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: elixir
+
+elixir: 
+  - '1.10'
+
+otp_release:
+  - '22.3'
```

Comment:
TIL about OTP! https://en.wikipedia.org/wiki/Open_Telecom_Platform

Reply:
For sure! I've been learning a lot about concurrency by learning Elixir, because Elixir was created to have first-class concurrency support, and building the language on top of the Erlang virtual machine was a key element of this.